### ISSUE TITLE
current showing 0:00 all the time

### ISSUE THREAD

**Issue by singh-saheb** [sarbagyastha/youtube_player_flutter#17]
the current duration does not gt update neither is the seekbar showing any update

**Comment by sarbagyastha** [comment]
which version are you on ?

**Comment by hadiisantoso** [comment]
I have this issue too,
currently I am using version : ^1.1.1+1

**Comment by DakotaSamuel** [comment]
Same here. Using 1.1.1+1. It doesn't happen on all videos. I would say ~30% of them. Video duration appears as 0:00 and pressing play does nothing.

![Screenshot_20190524-142518](https://user-images.githubusercontent.com/23041481/58352330-777d4200-7e28-11e9-9773-b7bff96f072d.jpg)

On Android, I was able to wait until the player was ready and then use controller.load() to get it to load. This does not work on IOS though. And the problem occurs on all videos. 

I am using the player in a pop-up page that appears when the user presses a button.
```dart
Navigator.push(
      context,
      new MaterialPageRoute(builder: (context) {
        //create the page
        return new Scaffold(
          body: Container(
            color: Colors.black,
            alignment: Alignment.center,
            child: YoutubePlayer(
              context: context,
              videoId: YoutubePlayer.convertUrlToId(widget.movie.trailer),
              autoPlay: true,
              showVideoProgressIndicator: true,
              videoProgressIndicatorColor: Theme.of(context).primaryColor,
              progressColors: ProgressColors(
                playedColor: Theme.of(context).primaryColor,
                handleColor: Theme.of(context).primaryColor,
              ),
              onPlayerInitialized: (controller) async {
                //check if we need to reload video
                if(controller.value.duration.inSeconds == 0) {
                  print("Video not loaded.");
                  //wait for the video player to be ready
                  print("Waiting for ready...");
                  while (!controller.value.isReady) {
                    print("Not ready.");
                    //wait a bit before checking again
                    await Future.delayed(Duration(milliseconds: 500));
                  }
                  print("Player ready!");
                  //reload
                  print("Reloading...");
                  controller.load();
                }
              },
            ),
          ),
        );
      }),
    )
```

**Comment by brunobrazio** [comment]
Same here

**Comment by tagerjump** [comment]
Same issue here. The process bar is not working for the same reason. The buffered part works just fine but the played part is always 0:00.

**Comment by tagerjump** [comment]
I think I found the problem. The microseconds increase by 1000 for one second. But one second equals to 10^6 second. 
![Screenshot_1559366470](https://user-images.githubusercontent.com/45743053/58744146-2c58c580-840c-11e9-8c84-709bd442151d.png)

**Comment by sarbagyastha** [comment]
The issue is fixed with [Version 1.2.0](https://pub.dev/packages/youtube_player_flutter#120). 
It was a silly mistake btw. 😜

**Comment by tagerjump** [comment]
Thank you for timely update. The package helps me a lot.




            
            
                
                    
                            
                                
                            
                            
                                123248001
                            
                    
                        
                            
                                    邮箱：123248001@qq.com
                            
                        
                
            
        Signature is customized by Netease Mail Master


On 06/03/2019 22:20, Sarbagya Dhaubanjar wrote: The issue is fixed with Version 1.2.0.
It was a silly mistake btw. 😜

—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or mute the thread.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/sarbagyastha/youtube_player_flutter/issues/17?email_source=notifications\u0026email_token=AK47XTJHMOZJVGWZSK24XGTPYXGOPA5CNFSM4HNU75MKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODW3GP6Q#issuecomment-498493434",
"url": "https://github.com/sarbagyastha/youtube_player_flutter/issues/17?email_source=notifications\u0026email_token=AK47XTJHMOZJVGWZSK24XGTPYXGOPA5CNFSM4HNU75MKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODW3GP6Q#issuecomment-498493434",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]
